src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -343,6 +343,12 @@
     <Compile Include="$(LinkerSourceFullPath)\linker\Mono.Linker.Steps\MarkStep.cs">
       <Link>Linker\Mono.Linker.Steps\MarkStep.cs</Link>
     </Compile>
+    <Compile Include="$(LinkerSourceFullPath)\linker\Mono.Linker\LoadException.cs">
```

Comment:
This is currently [breaking the build](https://jenkins.mono-project.com/job/xamarin-android-msbuild-pr-builder/445/console):

```
13:27:12   CSC : error CS2001: Source file '/Users/builder/jenkins/workspace/xamarin-android-msbuild-pr-builder/external/linker/linker/Mono.Linker/LoadException.cs' could not be found.
```

[linker/81007dfb does not contain a `LoadException.cs` there](https://github.com/mono/linker/tree/81007dfb0c48c1908d7540571d76fc7107128714/linker/Mono.Linker).